modules/3d_shapes_volume/include/3d_shapes_volume.h

```diff
@@ -0,0 +1,101 @@
+// Copyright 2021 Panova Olga
+
+#ifndef MODULES_3D_SHAPES_VOLUME_INCLUDE_3D_SHAPES_VOLUME_H_
+#define MODULES_3D_SHAPES_VOLUME_INCLUDE_3D_SHAPES_VOLUME_H_
+
+#include <cmath>
+
+class Shapes {
+ protected:
+    int _x, _y, _z;
+    double pi = 3.1415926535;
```

Comment:
Получение числа пи с большей точностью после запятой + константность
```suggestion
    const double PI = acos(-1.0);
```